### PR TITLE
gcs: attempt to fix title bar appearance on windows

### LINKED ISSUE #1349: dRonin logo doesn't play nicely with a black Windows task bar
- Seen in version (release version or githash): -
- On flight controller (if applicable): -

Issue details:
The letters are "invisible".
I believe @mlyle already has some updated icons, which include some white lining around the logo.

### PR DESCRIPTION
**Pull request by mlyle**
When the user uses a dark taskbar.

Fixes #1349.

### CODE REVIEW

**Comment by mlyle** [comment]
jenkins, build artifacts please

File: ground/gcs/src/plugins/coreplugin/coreconstants.h [hunk 1]
```diff
@@ -226,7 +226,7 @@ const char * const ICON_PLUGIN           = ":/core/images/pluginicon.png";
 const char * const ICON_EXIT             = ":/core/images/exiticon.png";
 const char * const ICON_OPTIONS          = ":/core/images/optionsicon.png";
 const char * const ICON_HELP             = ":/core/images/helpicon.png";
-const char * const ICON_GCS              = ":/core/gcs_logo_128";
+const char * const ICON_GCS              = ":/core/gcs_nontrans_64";
```
Comment (tracernz) [inline ground/gcs/src/plugins/coreplugin/coreconstants.h]: Not sure reducing this size is a good idea, some more modern things like 128px.

Comment (mlyle) [reply ground/gcs/src/plugins/coreplugin/coreconstants.h]: every other thing is setting the title bar icon to 32pix; (maybe i should change those to 64).  I don't have art for 128pix

**Comment by tracernz** [comment]
oldicon

newicon

**Comment by mlyle** [comment]
Ah so it actually fixens all of it.  neat.

**Comment by mlyle** [comment]
OK, a new variant to try.   Perhaps a bit more visually clean, more pixels, source art.

Jenkins, build artifacts please

**Comment by mlyle** [comment]
OK, I made the icon slightly better / use more of its permitted extents.  If/when this merges I'd appreciate 'squash and merge' so that we don't pollute the tree with all these intermediate states.

**Comment by tracernz** [comment]
Linux before and now:
 

Doesn't really fit the trend of transparent backgrounds, but looks pretty good anyway.

Windows before and now:

newnewicon
Decent improvement, Windows is a bit uglier though. :stuck_out_tongue:

**Pull request merged**